KnowledgeBases/TripAdvisor/TripAdvisor.py

```diff
@@ -105,8 +105,9 @@ def getReviews(keyword, place, entityType):
 
 	#check if there are some results specific to the city
 	#Idea in the world queries scope 1.
+	#Trying to support in the world queries
 	scope = int(soup('input', {'id': 'scope'})[0]['value'])
-	if scope == 1:
+	if scope == 1 and 'place' not in ['globe', 'world']:
```

Comment:
`'place' not in ['globe', 'world']` will always be true. I'm pretty sure you meant `some_str['place']`